packages/grid/_modules_/grid/components/menu/GridMenu.tsx

```diff
@@ -98,6 +98,7 @@ const GridMenu = (props: GridMenuProps) => {
       transition
```

Comment:
Could you add the slot for `Popper`? To keep each `components` key with a matching `componentsProps` key. I guess you need to pass the component using the `as` prop.

Reply:
Done